src/modules/listContainer.js

```diff
@@ -0,0 +1,39 @@
+import { base, gameId } from './env.js';
+
+const listContainer = () => {
+  const refresh = document.getElementById('refresh');
+  const listContainer = document.createElement('ul');
+  listContainer.className = 'all-scores';
+  let scoreArray = [];
+  const addToList = async () => {
+    const result = await fetch(`${base}${gameId}`).then((res) => res.json());
+    return result;
+  };
+  refresh.addEventListener('click', () => {
+    // addToList().then((res) => {
+    //   scoreArray = res.result;
+    //   scoreArray.forEach((score) => {
+    //     listContainer.innerHTML += `
+    //           <li> <p>${score.user}</p>
+    //        <p>${score.score}</p> </li> <hr>
+
+    //             `;
+    //   });
+    // });
+    window.location.reload();
```

Comment:
When clicking on the `Refresh` button the page shouldn't simply reload, it should make a `GET` call to the API to fetch all the scores and then update them in the UI. Please fix this.

<img width="658" alt="Screen Shot 2022-06-16 at 8 03 10" src="https://user-images.githubusercontent.com/82426682/174075690-26ad461a-09a0-4a9d-bd15-7b7931da1d71.png">